Orange/widgets/model/tests/test_owlogisticregression.py

```diff
@@ -112,3 +112,13 @@ def test_target_with_nan(self):
         self.send_signal("Data", table)
         coef2 = self.get_output("Coefficients")
         self.assertTrue(np.array_equal(coef1, coef2))
+
+    def test_class_weights(self):
+        table = Table("iris")
+        self.send_signal("Data", table)
+        self.assertFalse(self.widget.class_weight)
+        self.widget.controls.class_weight.setChecked(True)
+        self.assertTrue(self.widget.class_weight)
+        self.widget.apply_button.button.click()
+        self.assertEqual(self.widget.model.skl_model.class_weight, "balanced")
+        self.assertTrue(self.widget.Warning.class_weights_used.is_shown())
```

Comment:
I won't nitpick further, but the more important test, IMHO, is that the warning is cleared when necessary. We are good at raising them, but used to have problems clearing.

You also haven't tested that the argument is not set when the checkbox is unchecked. But let's assume that this but is unlikely, so I'm merging. :)